src/workers/atlas_strategies/atlas_challenge_resolution_strategy.js

```diff
@@ -9,6 +9,10 @@ This Source Code Form is “Incompatible With Secondary Licenses”, as defined
 
 /** @abstract */
 export default class AtlasChallengeParticipationStrategy {
+  get requiredFreeDiskSpace() {
+    return 50000000; // 50 MB
```

Comment:
50MB will be enough? What is the maximum bundle size?

Reply:
That's what strategies are for (not sure if anyone will change it though)